src/main/java/tp/cap5buddy/logic/commands/DeleteTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+package tp.cap5buddy.logic.commands;
+
+import tp.cap5buddy.contacts.ContactList;
+import tp.cap5buddy.modules.ModuleList;
+import tp.cap5buddy.todolist.Task;
+import tp.cap5buddy.todolist.TodoList;
+
```

Comment:
Perhaps javadocs could be added for the class here?

Reply:
Yes, I will add all the missing JavaDocs (I only added the javadocs that the CI requires for this PR)